backend/app/context/epoch_details.py

```diff
@@ -1,12 +1,14 @@
 from datetime import datetime
 from typing import List
 
+from app.settings import config
 from app.context.epoch_state import EpochState
 from app.context.helpers import check_if_future
 from app.extensions import epochs
 from app.infrastructure import graphql
 from app.infrastructure.external_api.etherscan.blocks import get_block_num_from_ts
 from app.legacy.utils.time import from_timestamp_s, sec_to_days
+from app.modules.common.blockchain_types import compare_blockchain_types, ChainTypes
```

Comment:
`context` package shouldn't be dependent on `module`, but only the other way around to avoid circular dependencies. Please, create a new `common` dir and put `blockchain_types` there

Reply:
I see, I guess we could add any references regarding the DDD way of structuring our project in future that you've implemented here. 

At a first glimpse, it looks like module with common is like a standard Pythonic package with general utils used by other project's domain.